.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Tools/WinML-Dashboard/deps/Netron"]
 	path = Tools/WinMLDashboard/deps/Netron
 	url = https://github.com/lutzroeder/Netron.git
+[submodule "Samples/RustSqueezenet/winrt-rs"]
```

Comment:
Why build a specific commit instead of using the winrt package?

Reply:
There are features that aren't released in the official package yet that are enabled in the latest version of the github repository. I explained it in the readme.md

Reply:
I'm asking winrt-rs folks, just to be sure.

Reply:
Yeah, without referencing the submodule, bind doesn't work as mentioned in the issue I linked.

Reply:
Issue filed: https://github.com/microsoft/winrt-rs/issues/318
It is a legit but acknowledge by winrt-rs team.